### ISSUE TITLE
Regression in 2.4.3/2.4.4: UnicodeDecodeError during install of sdist

### ISSUE THREAD

**Issue by johnthagen** [pyparsing/pyparsing#164]
Before pyparsing 2.4.4, our `pip download` commands were working properly, once 2.4.4 was released we saw this error in our builds.

Host OS: Ubuntu 18.04 x64
Python: Python 3.6

```
$ python3 -m pip download --no-binary=:all: -r requirements.txt
...
Collecting pyparsing>=2.0.2
  Downloading https://files.pythonhosted.org/packages/95/b7/1b8f7dbea6688a8ae97c6265e346120bb2a1888facaf58552ddbcbe502e3/pyparsing-2.4.4.tar.gz (644kB)
  Saved /tmp/tmpyjisa7uf/pyparsing-2.4.4.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /builds/.tox/bundle/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-download-d16rtszp/pyparsing/setup.py'"'"'; __file__='"'"'/tmp/pip-download-d16rtszp/pyparsing/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-download-d16rtszp/pyparsing/pip-egg-info
         cwd: /tmp/pip-download-d16rtszp/pyparsing/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-download-d16rtszp/pyparsing/setup.py", line 13, in <module>
        pyparsing_main_doc = README.read()
      File "/builds/.tox/bundle/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 212: ordinal not in range(128)
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by johnthagen** [comment]
Perhaps related to this commit? a94746d0526c0a01594938c52372fecb96d4941c

**Comment by twvd** [comment]
I noticed the same issue, see my PR here: https://github.com/pyparsing/pyparsing/pull/163

**Comment by johnthagen** [comment]
Should be fixed in #163. I'll wait for a new PyPI release and then close this issue, unless the maintainers want to close it now.

**Comment by ptmcg** [comment]
Fixed in 2.4.5 release
